toolkit/templates/big-stat.html

```diff
@@ -0,0 +1,7 @@
+{% macro big_stat(stat_value, stat_name, id=None) -%}
```

Comment:
Can this file be called `statistics.html` to match with its scss file? Seems to be the pattern we're using.